templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -137,6 +139,18 @@ objects:
           - name: asb-tls
             secret:
               secretName: asb-tls
+          - name: asb-auth-volume
+            secret:
+              secretName: asb-auth-secret
+
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: asb-auth-secret
+    namespace: ansible-service-broker
+  data:
+    username: YnJva2VyLWFkbWlu
```

Comment:
Let's introduce 2 params for the auth username/password in the parameters section

BASIC_AUTH_USER_NAME=YnJva2VyLWFkbWlu
BASIC_AUTH_USER_PASSWORD=TFl0SU5lc0hBZw==

Then refer to them here so it's easier to adjust the username/password on deploying.